src/nrtk_explorer/app/transforms.py

```diff
@@ -118,21 +124,89 @@ def delete_meta_state(old_ids, new_ids):
         self.state.transforms = [k for k in self._transforms.keys()]
         self.state.current_transform = self.state.transforms[0]
 
-        # Transform enabled control ##
-        self.state.transform_enabled = True
+        # Annotations enabled control via predictions_original_images_enabled ###
+        def update_prediction_original_images_enabled(**kwargs):
```

Comment:
why do you make them closure?

Reply:
I make the closure funcs to keep the state.change registration near the function definition.  Perhaps I should make a TrameApp class that extracts the feature enabled code?

Reply:
I was indeed thinking something like that

Reply:
I pulled out that code to a `ProcessingStep` class.